src/py42/services/audit_logs.py

```diff
@@ -0,0 +1,63 @@
+from py42 import settings
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.util import convert_datetime_to_timestamp_str
+
+
+class AuditLogsService(BaseService):
+    def get_page(self, page_num, page_size=None, **kwargs):
+
+        uri = u"/rpc/search/search-audit-log"
+        page_size = page_size or settings.items_per_page
+        params = dict(
+            page=page_num - 1,
+            pageSize=page_size,
+            dateRange=kwargs["date_range"],
+            eventTypes=kwargs["event_types"],
+            actorIds=kwargs["user_ids"],
+            actorNames=kwargs["user_names"],
+            actorIpAddresses=kwargs["user_ip_addresses"],
+            affectedUserIds=kwargs["affected_user_ids"],
+            affectedUserNames=kwargs["affected_user_names"],
+        )
+        params["type$"] = "audit_log::audit_log_queries.search_audit_log/1"
```

Comment:
Can you explain this line?  What is the /1 a the end mean?

Reply:
I think this is an oddity caused by some of the in-house code generation tools we use. Have we tried see if this request works without it? My guess is that it won't but I think this is a good example of API weirdness that we should avoid as an organization.

Reply:
This works without the `type$` field. Developed it based on `console.us.code42.com`. I will refer the swagger docs to check its significance.

Reply:
Removed.